### DISCUSSION TITLE
Install faceswaplab extension

### DISCUSSION THREAD

**Discussion by elyzionz** [MoonRide303/Fooocus-MRE#105] [Ideas]
it would be excellent if extensions could be installed as in automatic1111, for example, one of the extensions that is very good and would be great to be integrated is faceswaplab as it allows an excellent exchange of faces without the need to create loras.

**Comment by barepixels** [comment]
https://github.com/glucauze/sd-webui-faceswaplab

Only available for Vladmantic and a1111.  Not for Comfyui which Fooocus use
